src/reader.ts

```diff
@@ -4,11 +4,17 @@ import xss from "xss"
 function updatePage(){
     const article = JSON.parse(decodeURIComponent(atob(window.location.hash.substr(1))))
     article.content = xss(article.content, {stripIgnoreTag: true})
-    document.body.innerHTML = article.content
+    const content = document.createElement("main")
+    content.innerHTML = article.content
+    document.body.appendChild(content)
     if (article.title !== undefined) {
         const header = document.createElement("header")
         const title = document.createElement("h1")
-        document.title = [article.siteName, article.title].filter(Boolean).join(": ") // sensible?
+        if (article.link === undefined) {
+            document.title = [article.siteName, article.title].filter(Boolean).join(": ")
+        } else {
+            document.title = [article.siteName, article.title].filter(Boolean).join(": ") + " :: " + article.link
```

Comment:
Could we hide this behind a config setting by default? I don't think people would expect it

I need to check that the CSS changes don't change too much but otherwise this looks great :)